### PR TITLE
Add ModSecurity documentation to the README.md

### PR DESCRIPTION
**Pull request by mackwic**

### CODE REVIEW

**Comment by mackwic** [comment]
@EtienneM can you review our PR? WDYT ?

**Comment by Soulou** [comment]
Added @EtienneM  as reviewer

File: README.md [hunk 1]
```diff
@@ -106,19 +106,128 @@ server {
...
+# THEN block the request with a code 403 and log the event
+SecRule ARGS:param1 "@contains test" \
+	"id:1234,\
+	 deny,\
+	 log,\
+	 status:403,\
+	 severity: 'CRITICAL',\
+	 tag: 'custom-rule',\
+	 msg: 'this is the log message you will see',\
+	 logdata: '%{MATCHED_VAR_NAME}=%{MATCHED_VAR}'"
```
Comment (EtienneM) [inline README.md:200]: I don't understand well this part. Is it two different files to modify? If yes, I would create two code blocks.

Moreover having to create a nginx.conf.erb seems odd, shouldn't we make the ModSecurity rules file configurable via the environment with a default value set to `/app/vendor/nginx/conf/modsec-nginx.conf`?

**Review by EtienneM — COMMENTED** [review]
This doc is nice but shouldn't it be a documentation page on https://doc.scalingo.com/ rather than a section in the README?

**Comment by Soulou** [comment]
> This doc is nice but shouldn't it be a documentation page on https://doc.scalingo.com/ rather than a section in the README?

Both actually, since the buildpack is opensource it should be present here too I think.

File: README.md (hunk 1 above)
Comment (Soulou) [reply README.md:200]: Definitely, two different blocks, and it should NOT be of type `bash`.

Actually it's how it work for now @EtienneM the goal of the PR is to document what exists, we could make PR for improvement, today this way is the way to go.

This way doesn't replace ` /app/vendor/nginx/conf/modsec-nginx.conf` It adds an additional rule, the default Core Rule Set is still set and enabled.

**Pull request merged**